packages/gx-agent/src/agent/GXComplianceClient.ts

```diff
@@ -194,7 +198,7 @@ export class GXComplianceClient implements IAgentPlugin {
         // purpose: args.purpose,
         verifiableCredentials: [complianceCredential, serviceOffering.verifiableCredential],
         domain: did,
-        persist: true,
+        persist: args.persist ? args.persist : false,
```

Comment:
Not sure why this is needed everywhere, simply pass in args.persist

Reply:
changed to just: `args.persist`